### PR TITLE
Loop voice verification code until call is hung up.

### LINKED ISSUE #18: Make the push registration call voice repeat the number at least once
The 6 letter number you are told when using the "confirmation by call" for the push registration is only said once. Unless you know that beforehand and write it down quickly or if you have a good talent of quickly memorising the numbers you are in a bad situation and will probably need to toggle another call.

Please make the numbers be repeated at least once so slower people have a change of using that service successfully. Or announce them slower.

Originally posted at https://github.com/WhisperSystems/TextSecure/issues/1698
Closed as it was not the right place. If this is the wrong place again, please be so nice to use your superior knowledge of the project to copy this issue to the right place when closing it. Thanks!

### PR DESCRIPTION
**Pull request by patcon**
Closes #18.

### CODE REVIEW

File: src/main/java/org/whispersystems/textsecuregcm/sms/TwilioSmsSender.java [hunk 1]
```diff
@@ -40,7 +40,8 @@
 
   public static final String SAY_TWIML = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n" +
                                          "<Response>\n" +
-                                         "    <Say voice=\"woman\" language=\"en\">" + SmsSender.VOX_VERIFICATION_TEXT + "%s</Say>\n" +
+                                         "    <Say voice=\"woman\" language=\"en\">" + SmsSender.VOX_VERIFICATION_TEXT + "</Say>\n" +
+                                         "    <Say voice=\"woman\" language=\"en\" loop=\"3\">%s.</Say>\n" +
```
Comment (moxie0) [inline src/main/java/org/whispersystems/textsecuregcm/sms/TwilioSmsSender.java]: loop the whole thing

**Comment by patcon** [comment]
Done and squashed. The period was just to add a slight pause, as I don't think `<Pause>` can be inside a Say tag.

**Comment by patcon** [comment]
Just realized there's nexmo too, so hold on

**Comment by moxie0** [comment]
don't worry about nexmo

**Comment by patcon** [comment]
Ah oops.

I know it's not hooked up to the status API here, but I did sign the CLA fwiw

**Pull request closed without merging**